error_reporting/tests/system.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.cloud import error_reporting
+
+
+class TestErrorReporting(unittest.TestCase):
+
+    def test_report_exception(self):
+        client = error_reporting.Client()
+        try:
+            raise NameError
+        except NameError:
+            client.report_exception()
```

Comment:
@waprin This is insufficient.

You need to make a query to retrieve the reported error and then verify chunks of it (I don't know how difficult that is).

Also, let's push this branch to the https://github.com/GoogleCloudPlatform/google-cloud-python/ remote so that the system tests run (you can do that without closing this PR, but if you'd like to close it and open a new one from the branch in this remote, feel free)